editupdate.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+$db = new PDO('mysql:host=192.168.20.20;dbname=portfolio', 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$title = $_POST ['title'];
```

Comment:
no checks that these exist. can you do a check 
```
if (!empty($_POST['title']) && !empty($_POST['url'])) {
}
```

Reply:
this has not been resolved, please add the if checks like you did to the edit page